python/ray/experimental/dag/class_node.py

```diff
@@ -47,13 +72,14 @@ def __init__(self, actor: ClassNode, method_name: str):
         self._options = {}
 
     def _bind(self, *args, **kwargs):
+        kwargs_to_resolve = {
```

Comment:
Instead of calling this `kwargs_to_resolve` can we call it `other_args_to_resolve`